lib/earmark_parser/parser/link_parser.ex

```diff
@@ -87,6 +97,7 @@ defmodule Earmark.Parser.Parser.LinkParser do
 
   defp text_of_token(token)
   defp text_of_token({:escaped, text}), do: "\\#{text}"
+
```

Comment:
Sorry, the formatter kicked in on this file, but it should be all good still. :)